### PR TITLE
Add package termcap

### PR DESCRIPTION
**Pull request by h4waii**
Add termcap-1.3.1 package. This is sometimes a dependency for other packages.

### CODE REVIEW

File: packages/termcap/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+TERMUX_PKG_HOMEPAGE=https://www.gnu.org/software/termutils/
+TERMUX_PKG_DESCRIPTION="Termcap is a library and data base that enables programs to use display terminals in a terminal-independent manner."
+TERMUX_PKG_DEPENDS="termcap"
+TERMUX_PKG_VERSION=1.3.1
+TERMUX_PKG_SRCURL=https://ftp.gnu.org/gnu/termcap/termcap-${TERMUX_PKG_VERSION}.tar.gz
```
Comment (Auxilus) [inline packages/termcap/build.sh]: You can use http link instead of ftp `http://mirrors.kernel.org/gnu/termcap/termcap-${TERMUX_PKG_VERSION}.tar.gz`

Comment (h4waii) [reply packages/termcap/build.sh]: Just to clarify, it is served over HTTP(S), indicated by the protocol prefix.

Would you like the SRCURL changed to kernel.org and re-submitted?

Comment (Auxilus) [reply packages/termcap/build.sh]: Sure, all other gnu packages use `http://mirrors.kernel.org/gnu`

Comment (h4waii) [reply packages/termcap/build.sh]: Good to go. Thanks for requesting to match the same mirror for other GNU tools. Consistency is key!

**Comment by h4waii** [comment]
This doesn't actually install the linkable library for some reason. Closing, will re-opened one "massaging" is fixed.

**Comment by h4waii** [comment]
I think I can just re-open this to get my latest commits? Hope so!

**Comment by fornwall** [comment]
Thanks! I don't think using the old termcap library (last updated 2002) is the right way here. If we want a libtinfo.so we should build it from ncurses.

**Pull request closed without merging**